src/adagrams.js

```diff
@@ -1,15 +1,134 @@
+const poolHash = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+};
+
+const scoreChart = {
```

Comment:
This data structure looks nice, but isn't the most efficient.   In this project, the lookup will always start with the letter.  This structure makes it necessary to search for the letter in the arrays.  Using a data structure like poolHash would be more efficient.